UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -341,9 +367,12 @@ private async Task<bool> ProcessBatchFile(IEnumerable<string> uri, string downlo
                 }
                 else
                 {
+                    cancellationTokenSource.Cancel();
                     logger.LogError(EventIds.DownloadENCFilesNonOkResponse.ToEventId(), "Error in file share service while downloading ENC file:{fileName} with uri:{RequestUri} responded with {StatusCode} and BatchId:{BatchId} and _X-Correlation-ID:{correlationId}", fileName, httpResponse.RequestMessage.RequestUri, httpResponse.StatusCode, queueMessage.BatchId, queueMessage.CorrelationId);
+                    logger.LogError(EventIds.CancellationTokenEvent.ToEventId(), "Request cancelled while downloading ENC file:{fileName} from File Share Service with CancellationToken:{cancellationTokenSource.Token} at time:{DateTime.UtcNow} with uri:{RequestUri} responded with {StatusCode} and BatchId:{BatchId} and _X-Correlation-ID:{correlationId}", fileName, JsonConvert.SerializeObject(cancellationTokenSource.Token), DateTime.UtcNow, httpResponse.RequestMessage.RequestUri, httpResponse.StatusCode, queueMessage.BatchId, queueMessage.CorrelationId);
```

Comment:
Can "Request cancelled while downloading ENC file:{fileName} from File Share Service" be changed to "Request cancelled for Error in file share service while downloading ENC file:{fileName}" to maintain consistency across all logs of cancellation.